request/writer_test.go

```diff
@@ -0,0 +1,122 @@
+package request
+
+import (
+	"github.com/coredns/coredns/plugin/test"
+	"github.com/miekg/dns"
+	"reflect"
+	"testing"
+)
+
+func TestNewScrubWriter(t *testing.T) {
```

Comment:
This test is not needed.  `NewScrubWriter` is a one line function - too simple to warrant this.